armi/reactor/tests/test_blocks.py

```diff
@@ -1339,6 +1339,47 @@ def test_getMfp(self):
         assert_allclose(235.0, mfpAbs, rtol=0.1)
         assert_allclose(17.0, diffusionLength, rtol=0.1)
 
+    def test_getMgFlux(self):
+        applyDummyData(self.Block)
+        # These are unverified numbers, just the result of this calculation.
+        flux = self.Block.getMgFlux(average=True)
+        referenceFlux = [
+            3.68362183e07,
+            5.21203125e08,
```

Comment:
can these numbers be derived algorithmically from the dummy data loaded up so we don't have to manually rebuild this if that changes? Or at that point are we just re-implementing the function being tested?

Reply:
Probably? I'll look into it. I didn't dig too deep here, but it looks like it's just multiplying the flux and volume provided in the test.

Reply:
This test was actually not covering the if block that I wanted it to, so I'm going to delete it. We would need a block.p.lastMgFlux defined in order to test the part of getMgFlux() that I was targeting.